src/test/fuzz/strprintf.cpp

```diff
@@ -115,28 +115,50 @@ void test_one_input(const std::vector<uint8_t>& buffer)
         case 5:
             (void)strprintf(format_string, fuzzed_data_provider.ConsumeBool());
             break;
-        case 6:
+        }
+    } catch (const tinyformat::format_error&) {
+    }
+
+    if (format_string.find("%") != std::string::npos && format_string.find("c") != std::string::npos) {
+        // Avoid triggering the following:
+        // * strprintf("%c", 1.31783e+38);
+        // tinyformat.h:244:36: runtime error: 1.31783e+38 is outside the range of representable values of type 'char'
+        return;
+    }
+
+    if (format_string.find("%") != std::string::npos && format_string.find("*") != std::string::npos) {
```

Comment:
See my review comment above (matching logic and as micro-nit using `.find()`-version with `char` as first parameter).